src/main/java/com/gmail/nossr50/events/skills/rupture/McMMOEntityDamageByRuptureEvent.java

```diff
@@ -1,33 +1,23 @@
 package com.gmail.nossr50.events.skills.rupture;
 
 import com.gmail.nossr50.datatypes.player.McMMOPlayer;
-import com.gmail.nossr50.events.fake.FakeEntityDamageByEntityEvent;
+import com.google.common.collect.ImmutableMap;
 import org.bukkit.entity.Entity;
-import org.bukkit.entity.Player;
-import org.bukkit.event.entity.EntityDamageEvent;
+import org.bukkit.event.entity.EntityDamageByEntityEvent;
 import org.jetbrains.annotations.NotNull;
 
-import java.util.HashMap;
-import java.util.Map;
+import java.util.EnumMap;
 
-public class McMMOEntityDamageByRuptureEvent extends FakeEntityDamageByEntityEvent {
-	public McMMOEntityDamageByRuptureEvent(@NotNull Player damager, @NotNull Entity damagee, @NotNull DamageCause cause, double damage) {
-		super(damager, damagee, cause, getDamageMap(damage));
-	}
+public class McMMOEntityDamageByRuptureEvent extends EntityDamageByEntityEvent {
```

Comment:
Hmm, I'm not sure whether or not having it extend EntityDamageByEntityEvent is a good idea here, might cause unwanted behavior with other plugins.